api/src/scripts/load_dataset_on_create.py

```diff
@@ -75,6 +78,11 @@ def publish_all(feeds: List[Feed]):
     :param feeds: The list of feeds to publish
     """
     topic_path = get_topic_path()
+    logger.info(f"Publishing {len(feeds)} feeds to Pub/Sub topic {topic_path}...")
+    credentials, project = default()
+    logger.info(f"Authenticated project: {project}")
+    logger.info(f"Service Account Email: {credentials.service_account_email}")
     for feed in feeds:
+        logger.info(f"Publishing feed {feed.stable_id} to Pub/Sub topic {topic_path}...")
         publish(feed, topic_path)
```

Comment:
Can we follow the following pattern to increase performance and block the function from exiting before all messages are sent? https://github.com/googleapis/python-pubsub/blob/50504aed67b83c94d9c02fc8fbb58822bf8d1335/samples/snippets/publisher.py#L423

Reply:
done and tested ✅